app/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# [Choice] Node.js version (use -bullseye variants on local arm64/Apple Silicon): 18, 16, 14, 18-bullseye, 16-bullseye, 14-bullseye, 18-buster, 16-buster, 14-buster
```

Comment:
I think you have the changes from https://github.com/clicepfl/clic-website-v2/pull/38 here. Have you checked out from 37-add-nextjs-app-to-devcontainers instead of the current main? Consider rebasing so that we have a distinct set of changes 👍